week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,20 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(stringArr) {
+
+ // let arrayTidy = ["   /Daniel ", "  irina/ ", "     Gordon", "ashleigh "];
+  
+  for (i = 0; i<stringArr.length; i = i+1){
```

Comment:
Formatting this code to be neater would be best. But otherwise really good!

Reply:
Thanks Chris, I am clearing it for better.